### PR TITLE
feat: persist template fields

### PR DESCRIPTION
**Pull request by G3root**
[Screencast 2024-02-17 23:06:23.webm](https://github.com/opencapco/opencap.co/assets/84864519/a4616072-17f2-4f05-8ce1-a4590333afc6)

### CODE REVIEW

File: prisma/enum-generator.ts [hunk 1]
```diff
@@ -0,0 +1,39 @@
+/// credit : https://github.com/calcom/cal.com/blob/main/packages/prisma/enum-generator.ts
```
Comment (G3root) [inline prisma/enum-generator.ts:1]: to generate enum as const so we can use in `zod.nativeEnum`

File: .prettierignore [hunk 2]
```diff
@@ -1 +1,2 @@
-public/pdf.worker.min.js
\ No newline at end of file
+public/pdf.worker.min.js
+/src/prisma-enums/index.ts
```
Comment (G3root) [inline .prettierignore:2]: ignore generated file

**Comment by dahal** [comment]
@G3root looks like there are some merge conflicts. Please rebase it with current main.

**Review by dahal — APPROVED** [review]
🚀

**Pull request merged**